src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,45 @@
 import React from 'react';
+import './PostInfo.scss';
+import { Post } from '../../types/Post';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
-);
+type Props = {
+  post: Post;
+};
+
+export const PostInfo: React.FC<Props> = ({ post }) => {
+  const {
+    title,
+    body,
+    user,
+    comments,
+  } = post;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{title}</h3>
+
+        <p>
+          {' Posted by  '}
+
+          {user && <UserInfo user={user} />}
+        </p>
```

Comment:
let's avoid rendering if there is no user
```suggestion
{user && (
  <p>
    {' Posted by  '}
    <UserInfo user={user} />}
  </p>
)}
```

Reply:
or write something like `'Posted by Anonymous'`